lib/srv/uacc/uacc_stub.go

```diff
@@ -0,0 +1,31 @@
+// +build !linux
```

Comment:
No macOS support?

Reply:
Turns out Apple got rid of utmp/utmpx with Mac OS X 10.6 (Snow Leopard) and introduced the Apple Logging System for handling this data instead so there isn't much to support.

Reply:
I also lack a Mac for testing this stuff.

Reply:
I'd need to get a Mac development environment set up to for that which may take some time.

Reply:
Sure